pkg/constants/constants.go

```diff
@@ -0,0 +1,14 @@
+package constants
```

Comment:
i think this is missing the changes from https://github.com/openshift/cluster-logging-operator/pull/367